cctbx/sgtbx/__init__.py

```diff
@@ -829,8 +829,12 @@ def is_equivalent(O, rt_mx_ji):
     rs = rt_mx_ji.multiply(sso_j)
     return rs in O.registry
 
-def compare_cb_op_as_hkl(a, b): # FIXME Py2-style compare, wrap in functools.cmp_to_key
+def compare_cb_op_as_hkl(a, b):
+  # Deprecated. Do not use.
   if (len(a) < len(b)): return -1
   if (len(a) > len(b)): return  1
-  from past.builtins import cmp
-  return cmp(a, b)
+  if a < b:
+    return -1
+  elif a > b:
+    return 1
+  return 0
```

Comment:
Duplicate `cmp`
Change the import to `from libtbx.math_utils import cmp` so nothing else needs to be changed.